src/components/TopBar/components/Search/Search.scss

```diff
@@ -33,37 +38,30 @@ $search-results-desktop-large-width-offset: layout-width(secondary, min) +
 }
 
 .Overlay {
-  position: relative;
+  position: absolute;
+  left: 0;
+  right: 0;
   display: flex;
   flex-direction: column;
   align-items: stretch;
   width: 100%;
-
-  @include frame-when-nav-displayed {
-    width: calc(100% - #{$search-results-desktop-width-offset};);
-  }
-
-  @include breakpoint-after($large-width) {
-    width: calc(100% - #{$search-results-desktop-large-width-offset});
-  }
+  max-width: 70rem;
```

Comment:
Maybe store these numbers somewhere to make them a little less *magical*.

Reply:
I have extracted both values into variables and got them hoisted to the top of the file.